modules/cluster-logging-release-notes-5.4.0.adoc

```diff
@@ -0,0 +1,37 @@
+
+// Module included in the following assemblies:
+//cluster-logging-release-notes.adoc
+
+[id="cluster-logging-release-notes-5-4-0"]
+= {logging-title-uc} 5.4
+The following advisories are available for {logging} 5.4:
+//link:https://access.redhat.com/errata/ [{logging-title} Release 5.4]
+
+[id="openshift-logging-5-4-0-bug-fixes"]
+== Bug fixes
+*	Before this update, the cluster-logging-operator utilized cluster scoped roles and bindings to establish permissions for the prometheus service account to scrape metrics. These permissions were only created when deploying the Operator using the console interface but was missing when deploying from the command line. This fixes that issue by making this role and binding namespace scoped.	(link:https://issues.redhat.com/browse/LOG-2286[LOG-2286])
```

Comment:
Our current docs supports either of these two options: `cluster-logging-operator` or "Cluster Logging Operator" (capitalization varies). I lean toward `cluster-logging-operator` because that's what users see with `oc`.